docker-compose.dev.yml

```diff
@@ -1,9 +1,19 @@
 version: '3.4'
 
 services:
-  identifier:
+  web:
```

Comment:
as we disabled the basic auth for the other endpoints, it won't change anything for development on the frontend side. this is just for kalliope api

Reply:
but why are we starting a service that's not needed? That's the reason we split config in a .prod file

Reply:
identifier is not needed either. just open the port for resources and it will do the job. in that case kalliope needs it and it's part of the ticket resolution. you can always remove it later if u want, you can also create a docker-compose.sam.yml if an extra nginx service takes to much resources on your computer :/